apps/meteor/client/providers/CallProvider/CallProvider.tsx

```diff
@@ -161,20 +163,36 @@ export const CallProvider: FC = ({ children }) => {
 		roomCoordinator.openRouteLink('v', { rid });
 	}, []);
 
+	const findOrCreateVisitor = useCallback(
+		async (caller: ICallerInfo): Promise<ILivechatVisitor> => {
+			const phone = parseOutboundPhoneNumber(caller.callerId);
+
+			const { contact } = await getContactBy({ phone });
+
+			if (contact) {
+				return contact as unknown as ILivechatVisitor;
```

Comment:
This is something that is not exclusive to this PR, I seen this happen in other places, we need to investigate if this is an inconsistency issue between our types.

This is a work that will go over the scope of this PR so we can do this separately